ci/managedclustermodule-kmm-ci-build-sign.yaml

```diff
@@ -195,12 +83,12 @@ spec:
             baseImageRegistryTLS:
               insecure: true
             dockerfileConfigMap:
-              name: mod-example
+              name: kmm-kmod-dockerfile
```

Comment:
I don't think this is a good idea.

I though about it as well and tried it myself.
I don't think we should have any actual build as this part is already tested in the `e2e` job.

I think we should keep the dummy job to isolate as much as we can each workflow and reduce the duplicated parts as much as possible.

Reply:
But why duplicate keys and Dockerfile when we can reuse those of the e2e job?

Reply:
Regarding the keys, I agree.

Regarding the dockerfile, I have mixed feelings.
Re-using it will duplicate the building test across `e2e` and `e2e-hub`. I think it would be nice if the hub test is only testing the hub functionality in the manner of just checking that all the artifacts are created correctly but not actually have a real build or real.

With that said, I can also see benefits in re-using the current dockerfile. 

WDYT?

Reply:
I don't see any downside in re-using the same Dockerfile. Both tests having that part in common may make troubleshooting easier.